auem/data/datasets.py

```diff
@@ -0,0 +1,41 @@
+from typing import Union
+import pathlib
+import pandas as pd
+
+import torch
+import torchaudio
+from torchvision.transforms import Compose
+from torch.utils.data import Dataset
+
+
+class DCASE2020Task1a(Dataset):
```

Comment:
We may eventually want to make this a more genereic `DCASEDataset` class, if we find that their format is generally consistent.

But for now this is good